app/src/test/java/org/simple/clinic/newentry/PatientEntryScreenControllerTest.kt

```diff
@@ -555,18 +555,18 @@ class PatientEntryScreenControllerTest {
     verify(patientRepository).saveOngoingEntry(expectedSavedEntry)
   }
 
-  @Test // TODO: Migrate to Mobius
```

Comment:
I think we can add the `// TODO: Migrate to Mobius` line either above or below the `@Test` annotation instead of on the same line. This way, reading the diff becomes easier since it will only create a single-line change each time instead of two.

Reply:
This is for future PRs.

Reply:
This makes sense. Will do that for the following features.